templates/dashboard/includes/_address.html

```diff
@@ -1,13 +1,14 @@
 {% load format_address from i18n_address_tags %}
 {% load staticfiles %}
+{% load utils %}
 
 <div class="address">
 	<div class="formatted-address">{% format_address address latin=True %}</div>
 	{% if address.phone %}
 		<div>
 			<svg data-src="{% static "dashboard/images/phone.svg" %}" width="16" height="16" class="left" />
-			<a href="tel:{{ address.phone }}" class="phone-number">
-				{{ address.phone }}
+			<a href="tel:{{ address.phone|obfuscate_string }}" class="phone-number">
+				{{ address.phone|obfuscate_string }}
```

Comment:
As we introduced `phonenumber-field` phone numbers should have consistent format `+[country_code][phone_number]`, is leaving just a `+` sign enough? Shouldn't we leave at least few characters in that case so it's clear that's a phone number?